includes/abstracts/abstract-wc-legacy-product.php

```diff
@@ -375,4 +375,18 @@ public function get_cross_sells() {
 		_deprecated_function( 'WC_Product::get_cross_sells', '2.7', 'WC_Product::get_cross_sell_ids' );
 		return apply_filters( 'woocommerce_product_crosssell_ids', (array) maybe_unserialize( $this->crosssell_ids ), $this );
 	}
+
+	/**
+	 * Check if variable product has default attributes set.
+	 *
+	 * @access public
```

Comment:
I've been removing them too -- but missed that one when copy & pasting to the legacy class. I don't think they are necessary.